src/argilla/client/feedback/dataset/remote/base.py

```diff
@@ -280,10 +231,10 @@ def __init__(
         self._name = name
         self._workspace = workspace
 
-        self._records: RemoteFeedbackRecords = RemoteFeedbackRecords(dataset=self)
+        self._records = self.records_cls(dataset=self, **kwargs)
```

Comment:
We should let each implementation initialize its own records class.

Reply:
But then in the `RemoteFeedbackDatasetBase` we cannot include any method that relies on `_records` as we won't have those, and we would need to refine those twice